public/app/features/alerting/unified/rule-list/components/AlertRuleListItem.tsx

```diff
@@ -266,15 +266,17 @@ interface UnknownRuleListItemProps {
 
 export const UnknownRuleListItem = ({ rule, groupIdentifier }: UnknownRuleListItemProps) => {
   const styles = useStyles2(getStyles);
-  const { rulesSource, namespace, groupName } = groupIdentifier;
-
-  const ruleContext = {
-    name: rule.name,
-    groupName,
-    namespace: JSON.stringify(namespace),
-    rulesSource: rulesSource.uid === GrafanaRulesSourceSymbol ? GRAFANA_RULES_SOURCE_NAME : rulesSource.uid,
-  };
-  logError(new Error('unknown rule type'), ruleContext);
+
+  useEffect(() => {
```

Comment:
Why is this now wrapped in a useEffect?

Reply:
We don't want to log the error on each rerender. Only for the first time